templates/csi.yaml.j2

```diff
@@ -51,7 +50,7 @@ spec:
       serviceAccountName: kadalu-csi-nodeplugin-sa
       containers:
         - name: csi-node-driver-registrar
-          image: quay.io/k8scsi/csi-node-driver-registrar:v1.0.1
+          image: quay.io/k8scsi/csi-node-driver-registrar:v1.3.0
```

Comment:
the versions of csi sidecars are all different... any reason ?

Reply:
Each csi projects are in different repo with its own release numbers.

- csi-node-driver-registrar - https://github.com/kubernetes-csi/node-driver-registrar
- csi-attacher - https://github.com/kubernetes-csi/external-attacher
- csi-provisioner - https://github.com/kubernetes-csi/external-provisioner